agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/agogosml-ci-pipeline.yml

```diff
@@ -0,0 +1,126 @@
+resources:
+- repo: self
+
+variables:
+  container_registry: '{{cookiecutter.AZURE_CONTAINER_REGISTRY}}/'
+
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - '**/agogosml'
+    - '**/input_reader'
+    - '**/output_writer'
+
+jobs:
+- job: Phase_1
+  displayName: Agogosml-Build-CI
+  condition: succeeded()
+  pool:
+    vmImage: 'Ubuntu 16.04'
+
+  steps:
+  - task: Docker@1
+    displayName: 'Build an image'
+    inputs:
+      azureSubscriptionEndpoint: '{{cookiecutter.SUBSCRIPTION_ID}}'
+      azureContainerRegistry: {{cookiecutter.AZURE_CONTAINER_REGISTRY}}
+      dockerFile: '**/agogosml/Dockerfile.agogosml'
+      imageName: 'agogosml:$(Build.BuildId)'
+
+  - bash: |  
+       if [ $(Build.SourceBranchName) = "master" ]; then
+         echo "Since this is a merge build, we are creating a new 'latest' tag"
```

Comment:
the condition should follow docs here. [conditional tasks](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml#examples)
`and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))`

Reply:
Great suggestion; will change to conditional tasks.